packages/twenty-front/src/modules/ui/navigation/navigation-drawer/components/NavigationDrawerItem.tsx

```diff
@@ -233,28 +281,41 @@ export const NavigationDrawerItem = ({
         isNavigationDrawerExpanded={isNavigationDrawerExpanded}
         isDraggable={isDraggable}
       >
-        {showBreadcrumb && (
-          <NavigationDrawerAnimatedCollapseWrapper>
-            <NavigationDrawerItemBreadcrumb state={subItemState} />
-          </NavigationDrawerAnimatedCollapseWrapper>
-        )}
         <StyledItemElementsContainer>
+          {showBreadcrumb && (
+            <NavigationDrawerAnimatedCollapseWrapper>
+              <NavigationDrawerItemBreadcrumb state={subItemState} />
+            </NavigationDrawerAnimatedCollapseWrapper>
+          )}
+
           {Icon && (
-            <Icon
-              style={{ minWidth: theme.icon.size.md }}
-              size={theme.icon.size.md}
-              stroke={theme.icon.stroke.md}
-              color={
-                showBreadcrumb && !isSettingsPage && !isNavigationDrawerExpanded
-                  ? theme.font.color.light
-                  : 'currentColor'
-              }
-            />
+            <StyledIcon>
+              <Icon
+                style={{ minWidth: theme.icon.size.md }}
+                size={theme.icon.size.md}
+                stroke={theme.icon.stroke.md}
+                color={
+                  showBreadcrumb &&
+                  !isSettingsPage &&
+                  !isNavigationDrawerExpanded
+                    ? theme.font.color.light
+                    : 'currentColor'
+                }
+              />
+            </StyledIcon>
           )}
 
-          <NavigationDrawerAnimatedCollapseWrapper>
```

Comment:
we need that back

Reply:
@martmull I looked into it. But it was not possible. As I described in the description that having `NavigationDrawerAnimatedCollapseWrapper`  on top of any text or div won't let the truncation property work.

Reply:
seen with @Bonapara. We are ok to remove the animated effect